db/migrate/20190111110542_create_articles.rb

```diff
@@ -0,0 +1,13 @@
+class CreateArticles < ActiveRecord::Migration[5.2]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.text :lede
+      t.text :body
+      t.string :author
+      t.date :date
```

Comment:
What is the `date` attribute being used for?

Reply:
The date article is published

Reply:
That is NOT covered in this user story as far as I can see. Stick to the scope

Reply:
Fair enough, removed the date attribute. Using the model's time stamps instead.